### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#592] [Galaxy Trivia] [answered]
**LETISIONVE**
**MRAAEC**
**PPOLAT**
**PSEHRTAOMN**

**Comment by Veercodeprog** [comment]
The odd word out is "PSEHRTAOMN".

All of the other words ("LETISIONVE", "MRAAEC", and "PPOLAT") can be unscrambled to form valid English words:

"LETISIONVE" can be unscrambled to form "TELEVISION"
"MRAAEC" can be unscrambled to form "CAMERA"
"PPOLAT" can be unscrambled to form "LAPTOP"
However, "PSEHRTAOMN" cannot be unscrambled to form a valid English word. It appears to be a jumbled combination of letters without any clear meaning.

**Comment by peterbaccaro** [comment]
1) TELEVISION
2) CAMERA
3) LAPTOP
4) SMARTPHONE

The odd one out is **TELEVISION** because it is primarily used for displaying visual content and receiving broadcast signals, while CAMERA, LAPTOP, and SMARTPHONE are devices that are primarily used for capturing, processing, and storing visual content.

**Reply by ParanoidUser** [reply]
**Hint:** If we look at it from a power consumption perspective: Are there any differences between them?

**Reply by peterbaccaro** [reply]
From a power consumption perspective, a CAMERA, LAPTOP and SMARTPHONE are powered from batteries whereas a TELEVISION is powered by mains electricity.

On that basis the **TELEVISION** would still be the odd one out.

**Reply by ParanoidUser** [reply]
**Hint:** Hmm... interesting. One of them can actually be mechanical, i.e. "powered" by humans.

**Reply by peterbaccaro** [reply, marked as answer]
Ok, then you're referring to the fact that mechanical CAMERAS work without power.

On that basis the **CAMERA** is the odd one out since some do not need power.
Also all except CAMERA are communicating devices that can be used for talking or sending messages.

**Comment by thuongtruong1009** [comment]
The odd one out is "PSEHRTAOMN" because it is the only one that is an anagram of a longer word ("phantoms"). The other options are not anagrams of any other words.

"LETISIONVE" can be unscrambled to spell "television"
"MRAAEC" can be unscrambled to spell "camera"
"PPOLAT" can be unscrambled to spell "laptop"
